README.md

```diff
@@ -17,26 +17,25 @@ progress bar and optional information output as well.
 
 
 Code use example:
-```javascript
-var speedTest = require('speedtest-net');
+```js
+const speedTest = require('speedtest-net');
+const test = speedTest({maxTime: 5000});
 
-test = speedTest({maxTime: 5000});
-
-test.on('data', function(data) {
+test.on('data', data => {
   console.dir(data);
 });
 
-test.on('error', function(err) {
+test.on('error', err => {
   console.error(err);
 });
   
 ```
 
 Visual use example:
-```javascript
-var speedTest = require('speedtest-net');
+```js
+const speedTest = require('speedtest-net');
 
-speedTest.visual({maxTime: 5000}, function(err, data) {
+speedTest.visual({maxTime: 5000}, (err,data) => {
```

Comment:
Maybe, but this would need to be done throughout the code. I used to not do that but I've changed my ways since then. If you feel like doing it everywhere, then yes. Otherwise, just leave it consistent throughout.